shimmingtoolbox/read_nii.py

```diff
@@ -0,0 +1,84 @@
+import json
+import math
+import nibabel as nib
+import os
+
+
+def read_nii(niiPath):
+    """
+
+    Args:
+        niiPath (str): Path to the nifti file
+
+    Returns:
+        image:
+        info:
+        jsonData:
+    """
+
+    info = nib.load(niiPath)
+    image = info.get_fdata()
+
+    # `extractBefore` should get the correct filename in both.nii and.nii.gz cases
+    jsonPath = niiPath.split('.nii')[0] + '.json'
+
+    if os.path.isfile(jsonPath):
+        jsonData = json.load(open(jsonPath))
+    else:
+        jsonData = []
+        print('Warning: No json file has been found')
```

Comment:
```suggestion
        warnings.warn('No json file has been found')
```

or

```suggestion
```

because we can write our code to be able to deal with missing metadata.